railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb

```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Yes, and that line is at the top of every initializer, so I just added it for consistence with:

* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/wrap_parameters.rb.tt#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/session_store.rb.tt#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/mime_types.rb#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/inflections.rb#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/filter_parameter_logging.rb#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/cookies_serializer.rb#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/backtrace_silencers.rb#L1
* https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/initializers/assets.rb.tt#L1